CommandAPI/commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPICommand.java

```diff
@@ -50,6 +50,9 @@ public CommandAPICommand(String commandName) {
 		this.commandName = commandName;
 		this.isConverted = false;
 	}
+
+	//region With Commands (Adds/Subtracts traits)
```

Comment:
`//region` and `//endregion` is not a Java standard. Adding this in solely this one class introduces a complete inconsistency in code style with the rest of the project.